biome.jsonc

```diff
@@ -0,0 +1,375 @@
+              "crypto": "Please use node:crypto instead.",
+              "fs": "Please use node:fs instead.",
+              "http": "Please use node:https instead.",
+              "net": "Please use node:net instead.",
+              "os": "Please use node:os instead.",
+              "path": "Please use node:path instead.",
+              "stream": "Please use node:stream instead.",
+              "util": "Please use node:util instead.",
+              "url": "Please use node:url instead.",
+              "worker_threads": "Please use node:worker_threads instead."
+            }
+          }
+        },
+        "noDuplicateElseIf": "error",
+        "noUselessEscapeInRegex": "error",
+        "noIrregularWhitespace": "error",
+        "noOctalEscape": "error"
+      }
+    }
+  },
+  "javascript": {
+    "formatter": {
+      "jsxQuoteStyle": "double",
+      "quoteProperties": "asNeeded",
+      "trailingCommas": "es5",
+      "semicolons": "always",
+      "arrowParentheses": "always",
+      "bracketSpacing": false,
+      "bracketSameLine": false,
+      "quoteStyle": "double",
+      "attributePosition": "auto",
+      "enabled": true
+    },
+    "linter": {
+      "enabled": true
+    },
+    "globals": ["BigInt"]
+  },
+  "overrides": [
+    {
+      "include": ["packages/**/test/perf/**/*.test.ts", "packages/state-transition/test/utils/beforeValueMocha.ts"],
+      "javascript": {
+        // These are used by mocha
+        "globals": ["describe", "it", "before", "after"]
+      }
+    },
+    {
+      "include": ["packages/cli/src/", "packages/test-utils/src", "packages/flare/src"],
+      "linter": {
+        "rules": {
+          "suspicious": {
+            "noConsoleLog": "off"
+          }
+        }
+      }
+    },
+    {
+      "include": [
+        "**/*.config.js",
+        "**/*.config.mjs",
+        "**/*.config.cjs",
+        "**/*.config.ts",
+        "scripts/vitest/**/*.ts",
+        "scripts/vite/**/*.ts",
+        "**/types/**/*.ts",
+        "packages/api/src/beacon/routes/*.ts",
+        "packages/api/src/**/routes.ts",
+        "packages/api/src/utils/server/handler.ts",
+        "packages/api/test/unit/client/urlFormat.test.ts",
+        "packages/beacon-node/src/api/impl/config/constants.ts",
+        "packages/beacon-node/src/eth1/provider/eth1Provider.ts",
+        ""
+      ],
+      "linter": {
+        "rules": {
+          "style": {
+            "useNamingConvention": {
+              "level": "off",
+              "options": {
+                "strictCase": false
+              }
+            }
+          }
+        }
+      }
+    },
+    {
+      "include": [
+        "**/test/**/*.ts",
+        "packages/*/test/**/*.js",
+        "packages/api/src/utils/**/*.ts",
+        "packages/beacon-node/src/db/repositories/checkpointState.ts",
+        "packages/spec-test-util/src/single.ts"
+      ],
+      "linter": {
+        "rules": {
+          "suspicious": {
+            "noExplicitAny": "off"
```

Comment:
can't this be applied on per line basis?

Reply:
Yes we can disable by single line, but for these files we are overriding there are a lot of places where any is used. 

Unfortunately the only way to disable by files is these overrides, we can't do it with code comments.

Reply:
I see a lot of files in the diff where it's just a single line in a file where the eslint ignore is removed but no new ignore added, see https://github.com/ChainSafe/lodestar/pull/7108#discussion_r1792011665

Reply:
Those files would have been added in this override list. As few files were using single line comments, but had a lot of occurrences of same disable rules.